api/client/service/update.go

```diff
@@ -122,8 +117,8 @@ func updateService(spec *swarm.ServiceSpec, flags *pflag.FlagSet) error {
 	updateString(flagName, &spec.Name)
 	updateLabels(flags, &spec.Labels)
 	updateString("image", &cspec.Image)
-	updateSlice("command", &cspec.Command)
-	updateSlice("arg", &cspec.Command)
+	updateStringToSlice(flags, "command", &cspec.Command)
```

Comment:
So is the result of `--command "cat file1" --args "file2 file3"` the command line `cat file1 file2 file3`? If so, this concatenation seems a bit strange, and I wonder if we should disallow combinations like this (unless it's done this way for consistency with existing Docker behavior).